db/migrations/20220920084815_create_storeInfo_tables.sql

```diff
@@ -0,0 +1,12 @@
+-- migrate:up
+CREATE TABLE storeInfo (
```

Comment:
```suggestion
CREATE TABLE stores_infos (
```
stores 와 infos의 중간테이블이라면 다음과 같이 네이밍 해주시고 테이블이름에 대문자는 가급적 지양해주세요~